build.sh

```diff
@@ -31,28 +35,39 @@ cat <<EOF
 Usage: sh $0 command [argument]
 
 command:
-  download-core:        downloads core library (binary version)
   clean:                clean up/remove all generated files
-  build:            	builds iOS and OS X frameworks
-  ios-dynamic:     	builds iOS dynamic frameworks
-  ios-static:       	builds a fat iOS static framework
-  osx:              	builds OS X framework
-  test-ios:         	tests iOS framework
-  test-ios-devices: 	tests iOS on all attached iOS devices
-  test-osx:         	tests OSX framework
-  test:             	tests iOS and OS X frameworks
-  test-all:         	tests iOS and OS X frameworks with debug and release configurations
-  examples:         	builds all examples in examples/
-  browser:         	builds the Realm Browser OSX app
-  test-browser:     	tests the Realm Browser OSX app
-  verify:           	cleans, removes docs/output/, then runs docs, test-all, examples & browser
+  download-core:        downloads core library (binary version)
+  build:                builds all iOS  and OS X frameworks
+  ios-static:           builds fat iOS static framework
+  ios-dynamic:          builds iOS dynamic frameworks
+  ios-dynamic-fat:      builds fat iOS dynamic framework
```

Comment:
Sure, I'll remove it. If we choose to do #1595, we'll need to re-add it, but we should have that conversation first.